static/assets/svg-sprite.default.svg

```diff
@@ -1010,5 +1010,81 @@
+			<path id="path3626" style="fill:#000000;fill-opacity:1;fill-rule:nonzero;stroke:none" d="m 23.859,456.511 0,-0.047 -0.05,0 0.05,0.047 z"/>
+			<path id="path3628" style="fill:#000000;fill-opacity:1;fill-rule:nonzero;stroke:none" d="m 23.809,456.511 0.051,0 0,-0.047 -0.051,0 0,0.047 z"/>
+			<path id="path3630" style="fill:#000000;fill-opacity:1;fill-rule:nonzero;stroke:none" d="m 23.809,456.464 0,0.047 0.05,-0.047 -0.05,0 z m 0.05,0 -0.05,0.047 0,-0.047 0.05,0 z"/>
+			<path id="path3632" style="fill:#000000;fill-opacity:1;fill-rule:nonzero;stroke:none" d="m 23.859,456.464 -0.05,0 0.023,0 0,0.047 0.027,-0.047 z"/>
+			<path id="path3634" style="fill:#000000;fill-opacity:1;fill-rule:nonzero;stroke:none" d="m 23.809,456.464 0,0.047 0,-0.047 0.05,0.047 -0.05,-0.047 z"/>
+			<path id="path3636" style="fill:#000000;fill-opacity:1;fill-rule:nonzero;stroke:none" d="m 23.809,456.511 0.051,0 0,-0.047 -0.051,0 0,0.047 z"/>
+			<path id="path3638" style="fill:#000000;fill-opacity:1;fill-rule:nonzero;stroke:none" d="m 23.859,456.511 0,-0.047 -0.05,0 0.05,0.047 z"/>
+			<path id="path3640" style="fill:#000000;fill-opacity:1;fill-rule:nonzero;stroke:none" d="m 23.809,456.511 0.051,0 0,-0.047 -0.051,0 0,0.047 z"/>
+			<path id="path3642" style="fill:#000000;fill-opacity:1;fill-rule:nonzero;stroke:none" d="m 23.809,456.464 0,0.047 0.05,-0.047 -0.05,0 z m 0.05,0 -0.05,0.047 0,-0.047 0.05,0 z"/>
+			<path id="path3644" style="fill:#000000;fill-opacity:1;fill-rule:nonzero;stroke:none" d="m 23.859,456.464 -0.05,0 0.023,0 0,0.047 0.027,-0.047 z"/>
+			<path id="path3646" style="fill:#000000;fill-opacity:1;fill-rule:nonzero;stroke:none" d="m 23.809,456.464 0,0.047 0,-0.047 0.05,0.047 -0.05,-0.047 z"/>
+			<path id="path3648" style="fill:#000000;fill-opacity:1;fill-rule:nonzero;stroke:none" d="m 23.809,456.511 0.051,0 0,-0.047 -0.051,0 0,0.047 z"/>
+			<path id="path3650" style="fill:#000000;fill-opacity:1;fill-rule:nonzero;stroke:none" d="m 23.859,456.511 0,-0.047 -0.05,0 0.05,0.047 z"/>
+			<path id="path3652" style="fill:#000000;fill-opacity:1;fill-rule:nonzero;stroke:none" d="m 23.809,456.511 0.051,0 0,-0.047 -0.051,0 0,0.047 z"/>
+			<path id="path3654" style="fill:#000000;fill-opacity:1;fill-rule:nonzero;stroke:none" d="m 23.809,456.464 0,0.047 0.05,-0.047 -0.05,0 z m 0.05,0 -0.05,0.047 0,-0.047 0.05,0 z"/>
+			<path id="path3656" style="fill:#000000;fill-opacity:1;fill-rule:nonzero;stroke:none" d="m 23.859,456.464 -0.05,0 0.023,0 0,0.047 0.027,-0.047 z"/>
+			<path id="path3658" style="fill:#000000;fill-opacity:1;fill-rule:nonzero;stroke:none" d="m 23.809,456.464 0,0.047 0,-0.047 0.05,0.047 -0.05,-0.047 z"/>
+			<path id="path3660" style="fill:#000000;fill-opacity:1;fill-rule:nonzero;stroke:none" d="m 23.809,456.511 0.051,0 0,-0.047 -0.051,0 0,0.047 z"/>
+		</g>
+	</g>
+</symbol>
+<symbol id="icon-icon_roadworks-partial-closure" viewBox="0 0 600.99628 600.99628" height="600.99628" width="600.99628">
```

Comment:
As said above: oneway-closure describes it better

Reply:
Changed.